napari/_qt/layer_controls/qt_labels_controls.py

```diff
@@ -175,7 +175,7 @@ def __init__(self, layer) -> None:
 
         self.panzoom_button = QtModeRadioButton(
             layer,
-            'zoom',
+            'pan',
             Mode.PAN_ZOOM,
```

Comment:
I updated here `zoom` as it became totally misleading as the icon doesn't have anything to do with zoom. On the other hand, `PAN_ZOOM` is not misleading and describes the mode correctly (it pans and it can zoom). And if we care about backward compatibility changing the name of the mode may break other users' code, because it can be used by some users.

Reply:
Agreed that changing the mode name is a bigger and probably unnecessary disruption.

Reply:
For me lack of this change will introduce an inconsistency in code base and will require at least an open proper issue when merging this PR. 

Also, enum properly supports more than one name for a given value 

```
In [3]: from enum import Enum

In [4]: class Sample(Enum):
   ...:     one = 1
   ...:     two = 2
   ...:     new_two =2
   ...: 

In [5]: Sample.two == Sample.new_two
Out[5]: True
```

But adding deprecation warnings, unfortunately,  requires modifying metaclass...